examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1192 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": true
+   },
+   "source": [
+    "# 1 - Introduction to Acoustic Problem"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 1.1 - Introduction\n",
+    "\n",
+    "This notebook brings the description of the acoustic wave equation, which we also refer as the *acoustic problem*, \n",
```

Comment:
Please review the grammar here and in general throughout the series of notebooks. @ggorman says that grammarly (https://www.grammarly.com) is a nice tool for this.

Reply:
After all code corrections we will do a text revision and correct some text mistakes.